mobx_codegen/test/util_test.dart

```diff
@@ -37,9 +37,12 @@ MockMethod mockStreamMethod({
   bool isGenerator = false,
   bool returnsDynamic = false,
 }) {
-  final returnType = MockType();
-  // ignore: deprecated_member_use
-  when(() => returnType.isDynamic).thenReturn(returnsDynamic);
+  final DartType returnType;
+  if (returnsDynamic) {
+    returnType = DynamicMockType();
+  } else {
+    returnType = MockType();
+  }
```

Comment:
Shouldn't this be the following?

```suggestion
 final returnType = MockType();
  // ignore: deprecated_member_use
  when(() => returnType.isDynamic).thenReturn(returnsDynamic);
```

Reply:
![image](https://github.com/user-attachments/assets/ca2f298d-202c-4da2-bec6-3f27463e21ff)

We cant use isDynamic in newest version of dart, and should use "smth is DynamicType", so here we need create an instance of DynamicType if we want checks passed well.

Reply:
// ignore: deprecated_member_use 
sadly not prevent to fall in runtime